charmcraft/templates/init/metadata.yaml.j2

```diff
@@ -1,10 +1,25 @@
 # Copyright {{ year }} {{ author }}
 # See LICENSE file for licensing details.
 name: {{ name }}
+display-name: |
+  TEMPLATE-TODO: fill out a display name for the Charmcraft store
 description: |
   TEMPLATE-TODO: fill out the charm's description
 summary: |
   TEMPLATE-TODO: fill out the charm's summary
+tags: |
```

Comment:
This may not last much longer as a feature, it doesn't make a lot of sense for a charm developer to specify tags in this way imo. I'd rather not point people at this.